Include/Acidanthera/Library/OcStringLib.h

```diff
@@ -71,7 +71,7 @@ IsAsciiAlpha (
 
   @retval  TRUE, if character is a white space character.
 **/
-INTN
+BOOLEAN
```

Comment:
These are to match `isalpha` and `isspace` in `ctype.h`, aren't they?

Reply:
Hello, sorry, im not sure here whether this function should match standard ctype or not. Im just refer to function header comment that retval shoud return TRUE if matched.

Reply:
Yeah, better fix the header then. Non-zero if character is a white space character, Zero otherwise.

Reply:
Thanks for review & merging @PMheart @vit9696 
I will leave this commit & close this pull req to let you guys do for the best regarding this commit.
- peace -